projects/meanfinance/index.js

```diff
@@ -1,5 +1,56 @@
 const sdk = require("@defillama/sdk");
 const abi = require("./abi.json");
+const { request, gql } = require("graphql-request");
+
+const GRAPH_URLS = {
+  polygon: 'https://api.thegraph.com/subgraphs/name/mean-finance/dca-v2-polygon',
+  optimism: 'https://api.thegraph.com/subgraphs/name/mean-finance/dca-v2-optimism'
+}
+
+const DCA_HUB_ADDRESSES = {
+  polygon: '0x230C63702D1B5034461ab2ca889a30E343D81349',
+  optimism: '0x230C63702D1B5034461ab2ca889a30E343D81349'
+}
+
+const query = gql`
+  query tokens {
+    tokens {
+      id
+    }
+  }`
+;
+
+const getTokensInChain = async (chain) => {
+  try {
+    const result = await request(GRAPH_URLS[chain], query);
+    return result.tokens.map(({ id }) => id)
+  } catch (e) {
+    console.error(e);
+  }
+  return [];
+};
+
+function getV2TvlObject(chain) {
+  return {
+    tvl: (_, __, chainBlocks) => getV2TVL(chain, chainBlocks[chain])
+  }
+}
+
+async function getV2TVL(chain, block) {
+  const balances = {};
+  const tokens = await getTokensInChain(chain)
+  const hubAddress = DCA_HUB_ADDRESSES[chain]
+  for (const token of tokens) {
+    const balance = await sdk.api.erc20.balanceOf({
+      target: token,
+      owner: hubAddress,
+      block,
+      chain
+    })
+    sdk.util.sumSingleBalance(balances, `${chain}:${token}`, balance.output);
```

Comment:
[Suggestion]

`${chain}:${token}` this probably would work, but I can recommend fetching chain's transform function and using it

    const { getChainTransform } = require("../helper/portedTokens")